### PR TITLE
fixed: build without superlu

### PR DESCRIPTION
**Pull request by akva2**
Random discovery..

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: src/LinAlg/SparseMatrix.C [hunk 1]
```diff
@@ -1272,6 +1273,7 @@ bool SparseMatrix::solveSLUx (Vector& B, Real* rcond)
     if (rcond && slu)
       *rcond = slu->rcond;
   }
+#endif
 
   return true;
```
Comment (kmokstad) [inline src/LinAlg/SparseMatrix.C:1278]: Ok, this was due to the refactoring I did in b32d2cd9d159dab424099d6fe54ffbda64de4ad8. Another "error" introduced there is that this method is supposed to return `false` when no superlu, i.e., L178 should be replaced by
```
return ierr == 0;
```

Comment (akva2) [reply src/LinAlg/SparseMatrix.C]: ok.

**Comment by kmokstad** [comment]
you could put all these (#712 - #715) into one PR instead then, since they all are about gcc13 porting.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**